src/audit.ts

```diff
@@ -1,27 +1,41 @@
-const shimmer = require('shimmer')
-const assert = require('assert')
-const os = require('os')
-const uuidv4 = require('uuid/v4')
+import * as assert from 'assert'
+import * as os from 'os'
+import * as shimmer from 'shimmer'
+import * as uuidv4 from 'uuid'
+
+export interface IAudit {
```

Comment:
nit: a comment explaining what this is (or linking to the docs about it) might be nice